src/converters/piral-blazor/src/events.ts

```diff
@@ -67,8 +67,7 @@ function dispatchToRoot(event: any) {
     performInternalNavigation(event);
   }
 
-  // the mutation event cannot be cloned (at least in Webkit-based browsers)
-  if (!(event instanceof MutationEvent) && !event.processed) {
```

Comment:
I think in general this change is fine, but I'd make it such that it won't impact browsers that still supply it (which would still run in the other issue; that a mutationevent wants to be cloned).

--> My proposal

```js
if ((typeof MutationEvent === 'undefined' || !(event instanceof MutationEvent)) && !event.processed) {
  // ...
}
```